modules/ossm-vs-istio.adoc

```diff
@@ -39,6 +39,49 @@ The upstream Istio community installation automatically injects the sidecar into
 
 {ProductName} does not automatically inject the sidecar to any pods, but requires you to opt in to injection using an annotation without labeling projects. This method requires fewer privileges and does not conflict with other OpenShift capabilities such as builder pods. To enable automatic injection you specify the `sidecar.istio.io/inject` annotation as described in the Automatic sidecar injection section.
 
+[id="ossm-rbac_{context}"]
+== Istio Role Based Access Control features
+
+Istio Role Based Access Control (RBAC) provides a mechanism you can use to control access to a service. You can identify subjects by user name or by specifying a set of properties and apply access controls accordingly.
+
+The upstream Istio community installation includes options to perform exact header matches, match wildcards in headers, or check for a header containing a specific prefix or suffix.
+
+{ProductName} extends the ability to match request headers by using a regular expression. Specify a property key of `request.regex.headers` with a regular expression.
+
+.Upstream Istio community matching request headers example
+[source,yaml]
+----
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata: 
+  name: httpbin-usernamepolicy
+spec: 
+  action: ALLOW
+  rules: 
+    - when: 
+        - key: 'request.regex.headers[username]'
+          values: 
+            - "allowed.*"
+  selector: 
+    matchLabels: 
+      app: httpbin
+----
+
+.{ProductName} matching request headers by using regular expressions
+[source,yaml]
+----
+apiVersion: "rbac.istio.io/v1alpha1"
+kind: ServiceRoleBinding
+metadata:
+  name: httpbin-client-binding
+  namespace: httpbin
+spec:
+  subjects:
+  - user: "cluster.local/ns/istio-system/sa/istio-ingressgateway-service-account"
+    properties:
+      request.regex.headers[<header>]: "<regular expression>"
+----
```

Comment:
the ServiceRoleBinding resource is not required anymore. We used to have two resources, ServiceRole and ServiceRoleBinding that were required to do this, but now it's just AuthorizationPolicy, both upstream and in OSSM